model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -163,7 +163,7 @@ class LicenseInfoResolver(
                         licenseFinding.location,
                         appliedCuration = appliedCuration,
                         matchingPathExcludes = matchingPathExcludes,
-                        copyrights = resolvedCopyrightFindings.toSet()
+                        copyrights = resolvedCopyrightFindings
```

Comment:
Commit message: "to *it* reuse in another class"

Reply:
Done.